the-toad-tribune/src/layout/Politics.tsx

```diff
@@ -63,10 +77,43 @@ const PoliticsStyles = styled.div`
   background-color: grey;
   grid-area: 2/1/7/3;
   margin-left: 0.5em;
+  border-radius: 10px;
+  overflow: hidden;
 `;
 
 const Image = styled.img`
   height: 100%;
   width: 100%;
   overflow: none;
 `;
+
+const ImageContainer = styled.div`
+  max-height: 50%;
+  max-width: 100%;
+`;
+
+const PoliticsContentContainer = styled.div`
+  min-height: 50%;
+  min-width: 300px;
+  display: flex;
+  flex-direction: column;
+  margin: .4rem .6rem .6rem .6rem;
+`;
+
+const PoliticsTitle = styled.h3`
+  margin: .3rem;
+`;
+
+const PoliticsAuthors = styled.h4`
+  margin: .4rem;
+  
+`;
+
+const PoliticsPublishedDate = styled.h5`
+margin: 0 .4rem .2rem .4rem;
+`;
+
+const PoliticsDescription = styled.p`
+`;
```

Comment:
does this need to be here if there is no styling to it?

Reply:
The team likely intends to add styling at a later point.  When we do this, it's a good idea to add a `// TODO: Add styling`

Reply:
Thank you for the tips everyone!